app/App.js

```diff
@@ -168,7 +184,7 @@ export default class App extends Component {
             )
         }
         {
-          (localStorage[tokenKey] || localStorage[xcoobeeCookiesKey]) && (
+          (localStorage[tokenKey] || localStorage[xcoobeeCookiesKey]) && XcooBee.kit.config.testMode && (
```

Comment:
Is `testMode` a configuration option we need to publish in ReadMe Document.
What does it do?

Reply:
@Bilal-S `testMode` adds a refresh button for remove everything for local storage. It's just for easier testing.